releasenotes/notes/fix-unittest-git-patch-887a3963d4f58d8b.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    CI Visibility: Fixes `unittest` integration being patched without actually running any tests
```

```suggestion
    CI Visibility: fixes an issue where just importing `unittest` enabled CIVisibility and potentially caused unexpected logs and API requests 
```
Maybe something like that.